Dockerfile

```
@@ -6,23 +6,18 @@ ENV GO111MODULE auto
 
 RUN apt-get update && apt-get --no-install-recommends -y install \
     git \
-    unzip \
-    python3 \
-    python3-setuptools \
-    python3-pip \
-    python3-venv \
-    go-dep ;
+    unzip
```

```suggestion
    go-dep
    unzip
```

Just thought, [we still technically support go-dep in the entrypoint script](https://github.com/fac/terratest-github-action/blob/master/entrypoint.sh#L42) so we should probably keep it installed on the v1 release, despite it being deprecated and us not using it...